src/libaudtag/id3/id3v24.cc

```diff
@@ -664,6 +697,20 @@ bool ID3v24TagModule::write_tag (VFSFile & f, const Tuple & tuple)
     String comment = tuple.get_str (Tuple::Comment);
     add_comment_frame (comment, dict);
 
+    String lyrics = tuple.get_str (Tuple::Lyrics);
+    if (lyrics && lyrics[9])
```

Comment:
What's the deal with `lyrics[9]`? Was that meant to be `lyrics[0]`?

Reply:
I think this wast just a "keypress" error. The "0" and "9" keys are close to each other, and unfortunately I coppied this error from that project.